notebooks/Notebook1.md

```diff
@@ -18,17 +29,121 @@ exciting milestones, preliminary results, etc. Feel free to include images
+decision.
+
+I also decided to make it easy to reuse patterns. For example, suppose I had a
+rhythm that repeated every 16 measures. Rather than rewrite it everytime, I
+could save it to a variable and call it whenever I needed it after defining it
+only once. I decided to implement this using a markdown style implementation.
+
+```{"pattern 1"}
+du ga du ga du ga du ga
+don don tsu don
+```
+
+{"pattern 1"}
+
+The three backticks indicate that we're starting a "code block" where we define
+our rhythm. We give that rhythm a name (pattern 1, in this case) and pass it in
+as a string. Then, we can call it later using {patternName} syntax.
+
+Additionally, I added some tricks for reptition. Taking after a Pythonic syntax, you can
+represent repetition by using an asterisk.
+
+{"pattern 1"} * 4, for example, would repeat pattern 1 four times.
+
+This introduced the need for anonymous rhythms because what if you wanted to do
+repetition with a rhythm that wasn't repeated anywhere else? In that case, you
+could do something like this:
+
+{du ga du ga} * 4
+
+Which would repeat du ga du ga four times.
+
+In taiko, there are many times where people will split into different parts
+after playing in unison for a while. To remedy this, I tried doing something
+like the following:
+
+/// {A & B}
+
+--- {A}
+{pattern 1}
+---
+
+--- {B}
+{du ga du ga} * 4
+{ rest } * 4
+----
+
+///
+
+This would represent parts A and B, where perhaps half of the players would play
+pattern 1, and the other half would play the rhythm described in {B}. But then I
+want to edit it so it looks something like this instead, to resemble markdown
+more
+
+> "A & B"
+
+``` {> A}
+{pattern 1}
+```
+
+``` {> B}
+{du ga du ga} * 4
+{ rest } * 4
+```
+
+This also adds a sense of consistency with the fact that {name} means a rhythm
+block. The greater than sign typically means a blockquote in markdown, or a
+subset of a whole, so making this represent sections in my language makes sense
+to me. The > in front of the pattern name just distinguishes it from other
+rhythm blocks.
+
+In terms of other logistics, I decided to use Javascript for this project since
+that's what I'm familiar with. I also found a library that could be helpful in
+processing my plaintext and exporting it to a pdf: https://www.vexflow.com
+
+
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+I did > to represent different sections, but if I wanted to add other music
+notation like crescendo/decrescendo, using this symbol might conflict. It seems
+more intuitive that to represent that, I would use < or >. A solution could be to use
+>> and << instead...
+
+
 **What questions do you have for your critique partners? How can they best help
 you?**
 
+Reading through my plaintext representation of music, how does it feel to you?
```

Comment:
The syntax for splitting into two sections feels really verbose for me, but otherwise the project seems okay